xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -1148,7 +1167,8 @@
       <Docs>
         <param name="destination">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Writes an Int16 into a span of bytes as big endian.
```

Comment:
Same here for the Write methods. Add attempts to, add cref, fix summary and return tags formatting and add true and false conditions.

Reply:
Sigh... Whoever added the triple slash comments for these methods, put everything inside the `<summary></summary>`.